### PR TITLE
Follow the file structure of COLMAP

### PR DESCRIPTION
**Pull request by sarlinpe**

### CODE REVIEW

File: pycolmap/estimators/absolute_pose.cc [hunk 1]
```diff
@@ -95,8 +95,8 @@ py::object PyRefineAbsolutePose(
   return py::dict("cam_from_world"_a = refined_cam_from_world);
 }
 
-void bind_absolute_pose_estimation(py::module& m,
-                                   py::class_<RANSACOptions> PyRANSACOptions) {
+void BindAbsolutePose(py::module& m) {
```
Comment (Phil26AT) [inline pycolmap/estimators/absolute_pose.cc:98]: Why not BindAbsolutePoseEstimation? More explicit.

File: pycolmap/estimators/essential_matrix.cc [hunk 2]
```diff
@@ -92,7 +92,7 @@ py::object PyEstimateAndDecomposeEssentialMatrix(
                   "inliers"_a = ToPythonMask(inlier_mask));
 }
 
-void bind_essential_matrix_estimation(py::module& m) {
+void BindEssentialMatrix(py::module& m) {
```
Comment (Phil26AT) [inline pycolmap/estimators/essential_matrix.cc:95]: again: why not ...Estimation

**Comment by sarlinpe** [comment]
@Phil26AT done, please take a look.

**Review by Phil26AT — APPROVED** [review]

File: pycolmap/estimators/triangulation.cc [hunk 3]
```diff
@@ -41,8 +41,9 @@ py::dict PyEstimateTriangulation(
   return py::dict("xyz"_a = xyz, "inliers"_a = ToPythonMask(inlier_mask));
 }
 
-void bind_estimate_triangulation(py::module& m,
-                                 py::class_<RANSACOptions> PyRANSACOptions) {
+void BindTriangulation(py::module& m) {
```
[the remark this answers is not in the document]
Comment (sarlinpe) [reply pycolmap/estimators/triangulation.cc]: I'd stick to the structure of COLMAP. The incremental triangulation is at a higher logical level so it makes sense to keep it separate, no?

**Pull request merged**